### ISSUE TITLE
List of digitized Ethiopic manuscripts

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#553]
https://github.com/BetaMasaheft/Manuscripts/wiki/List-of-digitized-Ethiopic-manuscripts-available-online

**Comment by PietroLiuzzo** [comment]
this list has been started to put together sources of images of ethiopian manuscripts. it would be nice to tidy it up and provide also information on if they provide iiif access or not

**Comment by thea-m** [comment]
Dear all, 
it would be wonderful if, should new digitising projects come to your attention, you could update the list, write a short comment here or just let us know otherwise. We are trying to keep this list as complete as possible. 
Thank you!!

**Comment by Gnisci** [comment]
This is a lovely idea and you've already done an amazing job. For the BnF, it's much easier to go directly to: http://archivesetmanuscrits.bnf.fr/pageCollections.html?col=1 (I edited the Wiki link) which gives you an overview of all the digitized mms in the BnF, and you can click on Ethiopian ->Subfolders. It's much quicker!

**Comment by Gnisci** [comment]
Also: http://specialcollections.le.ac.uk/digital/collection/p15407coll6/id/19845

**Comment by thea-m** [comment]
Thank you, @Gnisci !

**Comment by Gnisci** [comment]
https://www.loc.gov/search/?fa=partof:manuscripts+in+the+libraries+of+the+greek+and+armenian+patriarchates+in+jerusalem:+microfilm+5017:+ethiopic&q=ethiopic&st=gallery

**Comment by thea-m** [comment]
Thank you, @Gnisci ! Added to the page.

**Comment by thea-m** [comment]
@anaiswion @martinaambu @AmelieChekroun @AlebachewBelay  @moldjira  @kimserge 
Here is the link to the [list of digitized Ethiopic manuscripts](https://github.com/BetaMasaheft/Manuscripts/wiki/List-of-digitized-Ethiopic-manuscripts-available-online) that we maintain. If you have anything to add, update or correct, that would be wonderful! You can edit the list directly, write your new information as a comment to this issue or send us an e-mail.
Thank you! Dorothea

**Comment by AlebachewBelay** [comment]
Ok, thanks Dorothea.

Best,

Alebachew

**Comment by thea-m** [comment]
As the list has been moved to the website, please write any new comments directly into this issue. Thank you very much for your contributions!

**Comment by thea-m** [comment]
Dear all,
This is a periodical reminder of the existence of this list of publicly available digitized Ethiopic manuscripts:
https://betamasaheft.eu/availableImages.html
Should you know of any publicly available digitized manuscripts that are not included there yet, it would be extremely kind to write a comment here or let us know by any other way, so that the list can be as complete as possible. We will also be happy to acknowledge you there.
Thank you!

@Sisay-SB @simseybo @SophiaD-M @Mersha-Alehegne @abausi @vitagrazia81 @SusanneHummel @antobrita @DenisNosnitsin1970 @Gnisci @sologebre @eu-genia @karljonaskarlsson @DariaElagina @anaiswion @nafisa-valieva

**Comment by karljonaskarlsson** [comment]
Hello! There are some samples of digitized manuscripts here, which maybe could be added: https://www2.lingfil.uu.se/projects/Dabra_SahelQ/ (click of "manuscripts" and you will arrive here: https://www2.lingfil.uu.se/projects/Dabra_SahelQ/MSS/DSMssSamples/).

**Comment by thea-m** [comment]
Thank you!

**Comment by SophiaD-M** [comment]
Pierpont Morgan Library in NY has the Zir Ganela Gospel online: https://www.themorgan.org/manuscripts/list?field_medieval_centuries_tid=All&field_image_available_tid=All&field_ms_type_tid=All&field_country_taxonomy_tid=19751

**Comment by thea-m** [comment]
Thank you!

**Comment by Gnisci** [comment]
Should the list not include just manuscripts that have been digitised in full? Because Zir Ganela, for instance, is just the illustrations.

**Comment by thea-m** [comment]
Yes, in principle that was the idea. We might add a section for partially digitised manuscripts (with even less hope for the list to be comprehensive) though, since they might still be useful and are often suggested.

**Comment by Gnisci** [comment]
A separate list would make a lot of sense, there is plenty of partially digitised material out there, and good to know about it, but it is of rather limited use for encoding purposes.

**Comment by SophiaD-M** [comment]
More more more! :) 

The National Library of Israel has several Ethiopian manuscripts, some 50 or so digitized online! They can also be downloaded freely. Unfortunately most descriptions are in Hebrew, but it is still a wonderful addition to this list.

http://aleph.nli.org.il/F/RJYQDTYTQPBBGFNXI65U4A7VDEFXQ61F4HM3R8MIDAVX89YCI8-07231?func=short-jump&jump=000001

**Comment by thea-m** [comment]
as pointed out by @SophiaD-M https://search.onb.ac.at/primo-explore/search?query=any,contains,%22Cod.%20Aethiop.%22,AND&tab=onb_digital&search_scope=ONB_digital&vid=ONB&mode=advanced&offset=0

**Comment by eu-genia** [comment]
I think the issue we have been using is https://github.com/BetaMasaheft/Documentation/issues/2126
